ddsreplayer/src/cpp/main.cpp

```diff
@@ -196,6 +186,36 @@ int main(
         // Load configuration from YAML
         eprosima::ddsrecorder::yaml::ReplayerConfiguration configuration(file_path);
 
+        if (log_verbosity.is_set() && !configuration.log_configuration.verbosity.is_set())
+        {
+            configuration.log_configuration.verbosity = log_verbosity;
+        }
+
+        if (log_filter.is_set() && !configuration.log_configuration.filter.is_set())
+        {
+            configuration.log_configuration.filter = log_filter;
+        }
+
+        if (!log_filter.is_set() && !configuration.log_configuration.filter.is_set())
+        {
+            configuration.log_configuration.filter.set_value({
+                {eprosima::utils::VerbosityKind::Error, ""},
+                {eprosima::utils::VerbosityKind::Warning, "(DDSREPLAYER|DDSPIPE)"},
+                {eprosima::utils::VerbosityKind::Info, ""}});
+            configuration.log_configuration.verbosity = eprosima::utils::VerbosityKind::Warning;
```

Comment:
Please don't set the verbosity here. It should already be set as the default value.